doc/esmvalcore/recipe.rst

```diff
@@ -77,24 +95,26 @@ For example, a datasets section could be:
 
 
 Note that this section is not required, as datasets can also be provided in the
-`Diagnostics`_ section.
+Diagnostics_ section.
 
+.. _Preprocessors:
 
-Preprocessors
-=============
+Recipe section: ``preprocessors``
+=================================
 
 The preprocessor section of the recipe includes one or more preprocesors, each
-of which may call one or several preprocessor functions.
+of which may execute one or several preprocessor steps.
 
 Each preprocessor section includes:
 
-- A preprocessor name.
-- A list of preprocesor functions to apply
-- Any Arguments given to the preprocessor functions.
-- The order that the preprocesor functions are applied can also be specified using the ``custom_order`` preprocesor function.
+- A preprocessor name (any name, one yaml tab (2 spaces) under ``preprocessors``);
```

Comment:
I think there is no need to specify the tabs here, this is part of the language syntax and applies everywhere through the recipe.